files/build_templates/buffers_config.j2

```diff
@@ -128,6 +128,16 @@ def
 {%- endfor %}
 {%- set port_names_active  = port_names_list_active  | join(',') %}
 
+{%- set port_names_list_extra_queues = [] %}
+{%- for port in PORT_ACTIVE %}
+    {% if (generate_queue_buffers_with_extra_lossless_queues is defined or generate_queue_buffers_with_extra_lossless_queues_with_inactive_ports is defined) and 
```

Comment:
are you able to access the macro as such? don't you need it to be defs.macro

Reply:
Yes, I'm able to access the macro because it's defined in `buffers.json.j2`. The macro is defined before `{%- include 'buffers_config.j2' %}`, so it's not in any namespace. We can access the macro in `buffers_config.j2` directly.

But now we have to add namespace `defs` to visit the macro `generate_queue_buffers_with_extra_lossless_queues ` as it's now defined in `buffers_default_tX.j2` now.